.circleci/config.yml

```diff
@@ -295,10 +295,10 @@ jobs:
       - run:
           name: Check build file for correct SDK version
           command: |
-            if grep -q "sdkVersion:\"$CIRCLE_TAG\"" mapbox-gl.js; then
-              echo SDK version in mapbox-gl.js matches $CIRCLE_TAG
+            if grep -q "sdkVersion:\"${CIRCLE_TAG:1}\"" ./dist/mapbox-gl.js; then
+              echo SDK version in mapbox-gl.js matches ${CIRCLE_TAG:1}
             else
-              echo SDK version in mapbox-gl.js does not match $CIRCLE_TAG
+              echo SDK version in mapbox-gl.js does not match ${CIRCLE_TAG:1}
               circleci-agent step halt
```

Comment:
I suggest replacing this halt by a red build, silently failing on that case will lead to forgetting cdn publish if not being thorough on the deployment steps.